react/SearchWrapper.tsx

```diff
@@ -282,14 +302,24 @@ const SearchWrapper: FC<SearchWrapperProps> = props => {
     query: { page },
     route: { title: pageTitle, metaTags },
   } = useRuntime() as RuntimeWithRoute
-  const settings = getSettings(APP_LOCATOR) || {}
+
+  const settings = getSettings(APP_LOCATOR) || ({} as StoreSettings)
   const loading = searchQuery ? searchQuery.loading : undefined
-  const { titleTag: defaultStoreTitle, storeName } = settings
+  const {
+    titleTag: defaultStoreTitle,
+    storeName,
+    enablePageNumberTitle = false,
+    canonicalWithoutUrlParams = false,
+    removeStoreNameTitle = false,
+  } = settings
+
   const title = getTitleTag({
     titleTag,
     storeTitle: storeName || defaultStoreTitle,
     term: params.term,
     pageTitle,
+    pageNumber: enablePageNumberTitle ? +page : 0,
```

Comment:
```suggestion
    pageNumber: enablePageNumberTitle ? page : 0,
```

Reply:
Maybe `Number(page)`?

Reply:
done, I applied Number(page), in this point we need convert page const to Number type